util/files.go

```diff
@@ -75,41 +75,6 @@ func ReadCertFile(fpath string) (*x509.Certificate, error) {
 	return cert, nil
 }
 
-func SecretKeyFromBytes(pemBytes []byte) (*ecdsa.PrivateKey, error) {
-	block, _ := pem.Decode(pemBytes)
-	return x509.ParseECPrivateKey(block.Bytes)
-}
-
-func X509PublicKeyBytesFromJWK(upkjwk jwk.Key) ([]byte, error) {
```

Comment:
I'm going to be using this writing to disk code fairly frequently. While I agree that we should be able to test without writing to disk, we should still keep the writing to disk functionality.

Reply:
e701e1c